### PR TITLE
Propogate DNS Zone Negative TTL down to CoreDNS

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

**Comment by k0da** [comment]
Requires https://github.com/k8gb-io/coredns-crd-plugin/pull/16

**Review by kuritka — DISMISSED** [review]

File: chart/k8gb/values.yaml [hunk 1]
```diff
@@ -11,6 +11,7 @@ k8gb:
   # but can be overrided with imageTag key
   # imageTag:
   dnsZone: "cloud.example.com" # dnsZone controlled by gslb
+  dnsZoneNegTTL: 3600 # Negative TTL for SOA record
```
Comment (ytsarev) [inline chart/k8gb/values.yaml:14]: why default negative is 3600 when it is clearly problematic got GSLB scenario? 
```suggestion
  dnsZoneNegTTL: 60 # Negative TTL for SOA record
```

**Review by ytsarev — CHANGES_REQUESTED** [review]

File: chart/k8gb/values.yaml (hunk 1 above)
Comment (k0da) [reply chart/k8gb/values.yaml]: What about we don't set any default value and keep plugins one, if set it will be propogated

Comment (ytsarev) [reply chart/k8gb/values.yaml]: @k0da i'm concerned that another customers will hit the same issue without having a clue that the root cause is actually negative TTL.  It is safer to keep it low imho

Comment (k0da) [reply chart/k8gb/values.yaml]: 10 minutes?

Comment (ytsarev) [reply chart/k8gb/values.yaml]: why?

Comment (k0da) [reply chart/k8gb/values.yaml]: IMO 1 minute is too agressive

Comment (k0da) [reply chart/k8gb/values.yaml]: agreed on 5 minutes

**Review by ytsarev — APPROVED** [review]
Looks great, thanks

**Review by somaritane — APPROVED** [review]

**Pull request merged**